modules/KalturaSupport/resources/mw.KCuePoints.js

```diff
@@ -302,29 +303,67 @@
 				_this.destroy();
 			});
 
+
 			// Bind to monitorEvent to trigger the cue points events and update he nextCuePoint
 			$(embedPlayer).bind(
 				"monitorEvent" + this.bindPostfix +
-					" seeked" + this.bindPostfix +
-					" onplay" + this.bindPostfix +
-					" KalturaSupport_ThumbCuePointsUpdated" + this.bindPostfix,
+				" seeked" + this.bindPostfix +
+				" onplay" + this.bindPostfix +
+				" KalturaSupport_ThumbCuePointsUpdated" + this.bindPostfix,
 				function (e) {
+					// TODO [es]: oren, we are we listening here for 'KalturaSupport_ThumbCuePointsUpdated'?
```

Comment:
@OrenMe I suggest you to review this function from the source directly (without seeing the lines removed) - [link to source](https://github.com/kaltura/mwEmbed/blob/webc-679/modules/KalturaSupport/resources/mw.KCuePoints.js#L315)